plugins/modules/zos_copy.py

```diff
@@ -916,6 +916,58 @@ def _merge_hash(self, *args):
             result.update(arg)
         return result
 
+    def file_has_crlf_endings(self, src):
+        """Reads src as a binary file and checks whether it uses CRLF or LF
+        line endings.
+
+        Arguments:
+            src {str} -- Path to a USS file
+
+        Returns:
+            {bool} -- True if the file uses CRLF endings, False if it uses LF
+                      ones.
+        """
+        with open(src, "rb") as src_file:
```

Comment:
This is not general knowledge and I could be overly analyzing here, but if a file is opened as `rb` in PYZ 3.8/3.9 the files are I believe tagged by PYZ if they are not. In other words, you are likely to get how ever they are tagged or UTF-8. In 3.10 or later PYZ, you got no tagging unless they are tagged. I don't know if this will have any impact, its probably best to put this in the interpreter and try a few files tagged and untagged. I did reach out to the SMEs on PYZ and asked them the same thing, feel free to comment if you have any other experience. 

Otherwise, this is good code and logic.

Reply:
For now, we can move forward and do some isolated testing while waiting to hear back from the SMEs who work on Python.